### PR TITLE
Update libri_trans asr recipe

### PR DESCRIPTION
**Pull request by hirofumi0810**
- Change RNN to Transformer
- Change character unit to BPE1k unit

### CODE REVIEW

**Comment by hirofumi0810** [comment]
@sw005320 I will update results with SpecAugment tomorrow.

**Comment by hirofumi0810** [comment]
@sw005320 Ready to be merged.

File: egs/libri_trans/asr1/conf/tuning/train_pytorch_transformer_bpe_long.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+maxlen-in: 512  # if input length  > maxlen-in, batchsize is automatically reduced
+maxlen-out: 150 # if output length > maxlen-out, batchsize is automatically reduced
+
+# optimization related
+sortagrad: 0 # Feed samples from shortest to longest ; -1: enabled for all epochs, 0: disabled, other: enabled for 'other' epochs
+opt: noam
+accum-grad: 2
+grad-clip: 5
+patience: 0
+epochs: 120
```
Comment (sw005320) [inline egs/libri_trans/asr1/conf/tuning/train_pytorch_transformer_bpe_long.yaml]: The only difference is the number of epochs? Then, we could just rename `train_pytorch_transformer_bpe_long.yaml ` as `train_pytorch_transformer_bpe.yaml` and replace it.

Comment (hirofumi0810) [reply egs/libri_trans/asr1/conf/tuning/train_pytorch_transformer_bpe_long.yaml]: I'd like to keep as they are in order to keep the training time for the baseline without SpacAugment.

Comment (sw005320) [reply egs/libri_trans/asr1/conf/tuning/train_pytorch_transformer_bpe_long.yaml]: Oh, now I got it. This longer version is for the specaug, right? Then, maybe better to add some comments in `run.sh`? Another suggestion is to use specaug as default in `run.sh`.

Comment (hirofumi0810) [reply egs/libri_trans/asr1/conf/tuning/train_pytorch_transformer_bpe_long.yaml]: I don't wanna use SpecAugment as the default setting. I added a comment in `run.sh`. Also, readers can easily follow it by looking at `RESULTS.md`.

Comment (sw005320) [reply egs/libri_trans/asr1/conf/tuning/train_pytorch_transformer_bpe_long.yaml]: ok. thanks!

**Comment by sw005320** [comment]
LGTM

**Pull request merged**